build.sh

```diff
@@ -15,5 +15,5 @@ rm -rf manifests
 mkdir -p manifests/setup
 
                                                # optional, but we would like to generate yaml, not json
```

Comment:
Nit: Can you fix the indent here, thanks! :)

Reply:
I believe this was meant to apply to the call to `gojsontoyaml`, I have updated the indentation and the comment to reflect that.